go.mod

```diff
@@ -11,7 +11,7 @@ replace (
 require (
 	github.com/google/go-cmp v0.5.9
 	github.com/projectcontour/contour v1.25.0
-	go.uber.org/zap v1.25.0
+	go.uber.org/zap v1.26.0
```

Comment:
you'll have to roll this change back to v1.25.0

or run `./hack/update-deps` script